validate.lic

```diff
@@ -88,6 +88,12 @@ class DRYamlValidator
             .each { |skill_name| error("Invalid weapon_training: skill name '#{skill_name}'. Valid skills are '#{@valid_weapon_skills}'") }
   end
 
+  def assert_that_dance_skill_is_skill(settings)
```

Comment:
💡 What do you think about making this name a bit more specific: `...dance_skill_is_melee_skill`?